tests/unit/apps/test_auxiliary.py

```diff
@@ -1151,7 +1160,16 @@ def test_ovn_check_version_pinning_version_pinning_config_True(model):
         },
         workload_version="22.03",
     )
-    exp_msg = f"Cannot upgrade '{app.name}'. 'enable-version-pinning' must be set to 'false'."
+    exp_msg = (
```

Comment:
I’m fine with removing the message checks in the unit tests. 
It's unnecessary and annoyed to verify the entire message content.

Reply:
(non-blocking, free to resolve)

For me it's because the ApplicationError is too vague. How about create a new Error and put the messge there?

Reply:
My suggestion would be to retain the message check, but reduce it to a regex like `.*enable-version-pinning.*`, to at least assert that the error is related to the version pinning. Otherwise it could be crashing for another reason and the tests would not know.

Reply:
- new error would be the best but highest cost
- regex would be good but still not solid
- removing it would be the easy but not good

It's fair to choose the middle one 👍🏼

Reply:
> If the message content is important, I’d prefer to match specific keywords, which should be sufficient.

Done